src/components/panel/panel.js

```diff
@@ -438,10 +437,11 @@ angular
 
 /**
  * @ngdoc method
- * @name MdPanelPosition#withPanelXPosition
- * @param {string} xPosition
+ * @name MdPanelPosition#withPanelPosition
+ * @param {string} xpos
```

Comment:
nit: Can you keep these as `xPosition` and `yPosition` as it directs the user towards the constants with the same name?